index.html

```diff
@@ -12,6 +12,9 @@
 <body>
   <header><h1>GrumpyGram</h1></header>
   <div class="container">
+    <div>
+      Adding text
```

Comment:
@jko113 this looks pretty good, but you forgot the exclamation points. We need 3. It's standard.

Reply:
!!! like this?

Reply:
@radishmouse where are you?